src/main/java/seedu/address/logic/parser/AddMemberCommandParser.java

```diff
@@ -14,16 +12,15 @@ public class AddMemberCommandParser implements Parser<AddMemberCommand> {
     /**
      * Parses the given {@code String} of arguments in the context of the AddMemberCommand
      * and returns a AddMemberCommand object for execution.
+     *
      * @throws ParseException if the user input does not conform to the expected format
      */
     public AddMemberCommand parse(String args) throws ParseException {
-        try {
-            Index index = ParserUtil.parseIndex(args);
-            return new AddMemberCommand(index);
-        } catch (ParseException pe) {
-            throw new ParseException(
-                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddMemberCommand.MESSAGE_USAGE), pe);
+        String name = args.trim();
+        if (!Name.isValidName(name)) {
+            throw new ParseException(Name.MESSAGE_CONSTRAINTS);
```

Comment:
Should we also leave the `MESSAGE_INVALID_COMMAND_FORMAT` as well?

Reply:
I'm not too sure what you mean

Reply:
```suggestion
            throw new ParseException(
                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, Name.MESSAGE_CONSTRAINTS));
```

though i'm not sure which one makes more sense, either ways just a tiny nitpick